Add a section about custom resource tags (#10260)
DataDog/documentation: content/en/infrastructure/livecontainers.md

{{% /tab %}}
{{< /tabs >}}

### Add custom tags to resources

> It is possible to add custom tags to Kubernetes resources to ease filtering inside the Kubernetes resource view.

Reviewer: ```suggestion
You can add custom tags to Kubernetes resources to ease filtering inside the Kubernetes resource view.
```

Additional tags are added through the `DD_ORCHESTRATOR_EXPLORER_EXTRA_TAGS` environment variable.

> Note that such tags will only show up in the Kubernetes resource view.

Reviewer: ```suggestion
**Note**: These tags only show up in the Kubernetes resource view.
```
